package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "soos-dast",
+  "version": "2.0.0",
+  "description": "SOOS DAST - The affordable no limit web vulnerability scanner",
+  "main": "index.js",
+  "scripts": {
+    "setup:install": "npm install",
+    "setup:clean-install": "npm ci",
+    "setup:update": "npx npm-check -u",
+    "setup:clean": "npx rimraf node_modules && npx rimraf package-lock.json",
+    "build": "tsc",
+    "build:clean": "npx rimraf build",
+    "format": "prettier ./src --check",
+    "format:fix": "prettier ./src --write",
+    "typecheck": "tsc --noEmit",
+    "test:coverage": "npm run test -- --reporter xunit --reporter-option output=ResultsFile.xml",
+    "check": "npm run format && npm run typecheck && npm run test"
```

Comment:
if we don't have ts tests yet, remove npm run test from the check - also add `&& npm run outdated`